css/shepherd/shepherd-theme-square.css

```diff
@@ -0,0 +1,248 @@
+      .shepherd-element.shepherd-theme-square .shepherd-content header h3 {
+        margin: 0;
+        line-height: 1;
+        font-weight: normal; }
+      .shepherd-element.shepherd-theme-square .shepherd-content header a.shepherd-cancel-link {
+        float: right;
+        text-decoration: none;
+        font-size: 1.25em;
+        line-height: .8em;
+        font-weight: normal;
+        color: rgba(0, 0, 0, 0.5);
+        opacity: 0.25;
+        position: relative;
+        top: .1em;
+        padding: .8em;
+        margin-bottom: -.8em; }
+        .shepherd-element.shepherd-theme-square .shepherd-content header a.shepherd-cancel-link:hover {
+          opacity: 1; }
+    .shepherd-element.shepherd-theme-square .shepherd-content .shepherd-text {
+      padding: 1em; }
+      .shepherd-element.shepherd-theme-square .shepherd-content .shepherd-text p {
+        margin: 0 0 .5em 0;
+        line-height: 1.3em; }
+        .shepherd-element.shepherd-theme-square .shepherd-content .shepherd-text p:last-child {
+          margin-bottom: 0; }
+    .shepherd-element.shepherd-theme-square .shepherd-content footer {
+      padding: 0 1em 1em; }
+      .shepherd-element.shepherd-theme-square .shepherd-content footer .shepherd-buttons {
+        text-align: right;
+        list-style: none;
+        padding: 0;
+        margin: 0; }
+        .shepherd-element.shepherd-theme-square .shepherd-content footer .shepherd-buttons li {
+          display: inline;
+          padding: 0;
+          margin: 0; }
+          .shepherd-element.shepherd-theme-square .shepherd-content footer .shepherd-buttons li .shepherd-button {
+            display: inline-block;
+            vertical-align: middle;
+            *vertical-align: auto;
+            *zoom: 1;
+            *display: inline;
+            border-radius: 0;
+            cursor: pointer;
+            border: 0;
+            margin: 0 .5em 0 0;
+            font-family: inherit;
+            text-transform: uppercase;
+            letter-spacing: .1em;
+            font-size: .8em;
+            line-height: 1em;
+            padding: .75em 2em;
+            background: #3288e6;
+            color: #fff; }
+            .shepherd-element.shepherd-theme-square .shepherd-content footer .shepherd-buttons li .shepherd-button.shepherd-button-secondary {
+              background: #eee;
+              color: #888; }
+          .shepherd-element.shepherd-theme-square .shepherd-content footer .shepherd-buttons li:last-child .shepherd-button {
+            margin-right: 0; }
+
+.shepherd-start-tour-button.shepherd-theme-square {
+  display: inline-block;
+  vertical-align: middle;
+  *vertical-align: auto;
+  *zoom: 1;
+  *display: inline;
+  border-radius: 0;
+  cursor: pointer;
+  border: 0;
+  margin: 0 .5em 0 0;
+  font-family: inherit;
+  text-transform: uppercase;
+  letter-spacing: .1em;
+  font-size: .8em;
+  line-height: 1em;
+  padding: .75em 2em;
+  background: #3288e6;
+  color: #fff; }
```

Comment:
Somehow it doesn't feel quite right to include these vendor libs in the repo. Maybe we could point to unpkg CDN instead?